test/api.test.ts

```diff
@@ -0,0 +1,53 @@
+import fastify from 'fastify';
+import { Server, IncomingMessage, ServerResponse } from 'http';
+import * as jest from 'jest';
```

Comment:
Let's use bun:test instead of jest.

```js
import { test, expect, mock } from "bun:test";
const random = mock(() => Math.random());

test("random", async () => {
  const val = random();
  expect(val).toBeGreaterThan(0);
  expect(random).toHaveBeenCalled();
  expect(random).toHaveBeenCalledTimes(1);
});
```